lighthouse-core/computed/metrics/cumulative-layout-shift.js

```diff
@@ -40,10 +39,7 @@ class CumulativeLayoutShift {
       // For all-frames CLS calculation, we rely on `weighted_score_delta`, which
       // was added in Chrome 90: https://crbug.com/1173139
```

Comment:
nit
```suggestion
      // For all-frames CLS calculation, we rely on `weighted_score_delta`
      // All layout shift events should have this since M90: https://crbug.com/1173139
```